{{cookiecutter.github_repository}}/provisioner/roles/nginx/templates/site.443.conf.j2

```diff
@@ -44,6 +44,8 @@ server {
     location /docs {
         # mkdocs html generated via "product_data" role
         alias {{ project_path }}/_docs_html/;
+        auth_basic "Restricted Content";
+        auth_basic_user_file {{ htpasswd_file_path }};
```

Comment:
How you introducing another variable `protected_docs`? then use that flag to set the protection. Do always docs need to be protected by everyone, this variable just makes the whole thing simple. @tucosaurus